include/glow/Graph/Graph.h

```diff
@@ -243,6 +243,11 @@ class Function final : public Named {
                              llvm::ArrayRef<unsigned_t> strides,
                              llvm::ArrayRef<unsigned_t> pads);
 
+  MaxPoolNode *createMaxPool(llvm::StringRef name, NodeValue input,
```

Comment:
since we enforce output type to be the same as input type for maxPool, i think we can just use `createMaxPool` method above (no need to overload it).

Reply:
updated. Thanks!